Rivet/Functions/Update-Database.ps1

```diff
@@ -142,14 +143,14 @@ function Update-Database
         Get-MigrationFile -Path $Path -Configuration $Configuration @byName -ErrorAction Stop |
         Sort-Object -Property 'MigrationID' -Descending:$popping |
         Where-Object {
-            if( $RivetSchema )
+            if( $RivetSchema -or ($_.BaseName -eq 'schema') )

```

Comment:
Just use `if ($RivetSchema)`.

Reply:
If I don't check for the schema file here, it will throw an error in the next check `if( [int64]$_.MigrationID -lt $script:firstMigrationId )` and the schema file ID would be less than the first migration ID, throwing an error.

Reply:
Using `$RivetSchema` should skip that check.

Reply:
`Initialize-Database` calls `Update-Database` with the `-RivetSchema` switch when pushing the schema.ps1 file, so `if ($RivetSchema)` should return true and you should never get to the check on line 151.